__tests__/test.js

```diff
@@ -66,3 +69,27 @@ describe("test DB for requests", () => {
       });
   });
 });
+
+describe("GET /api/articles ", () => {
+  test("/api/articles returns all article in order of date", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((res) => {
+        expect(res.status).toBe(200);
+        expect(res.body.length).toBeGreaterThan(0);
+        expect(res.body).toBeSorted("comments.created_at");
+        expect(res.body).toBeSorted({ descending: true });
```

Comment:
Your order assertion should be one line. If you look at the jest sorted docs again https://www.npmjs.com/package/jest-sorted you can see that the default sorting will be ascending. So technically line 80 and line 81 should be directly contradicting each other. 

My theory is that line 80 isn't actually doing a valid assertion because the returned arr of objects do not have a 'comments.created_at' property. What I want you to do is remove line 80 completely and make sure line 81 is asserting the right thing (that we're sorting in descending order by the .created_at property)